specs/linux/selinux_settings.table

```diff
@@ -0,0 +1,11 @@
+table_name("selinux_settings")
+description("Track active SELinux settings.")
+schema([
+    Column("scope", TEXT, "Scope", index=True),
+    Column("key", TEXT, "Key", index=True),
+    Column("value", TEXT, "Value"),
```

Comment:
Elaborating on what to expect for the value is helpful, for example "boolean values are either on or off"